DB/oci8.php

```diff
@@ -383,14 +383,24 @@ function fetchInto($result, &$arr, $fetchmode, $rownum = null)
             return $this->raiseError(DB_ERROR_NOT_CAPABLE);
         }
         if ($fetchmode & DB_FETCHMODE_ASSOC) {
-            $moredata = @oci_fetch_array($result,$arr,OCI_ASSOC+OCI_RETURN_NULLS+OCI_RETURN_LOBS);
+            $arr = @oci_fetch_array($result,OCI_ASSOC+OCI_RETURN_NULLS+OCI_RETURN_LOBS);
```

Comment:
While silence?

Reply:
i think, because of all other PHP oci functions are silent, too. Maybe the DBMS should handle the error messages on its own?
There is also another check, if the `oci_fetch_array` returns an array and not false.
i don´t have a strong opinion on call it silence or not.
@ashnazg is that right? 
@ashnazg @till what do you suggest?

Reply:
@schengawegga No, that sounds plausible. Otherwise LGTM!